app/models/concerns/spotlight/solr_document/atomic_updates.rb

```diff
@@ -1,14 +1,19 @@
 module Spotlight::SolrDocument::AtomicUpdates
 
-  def reindex
-    data = hash_for_solr_update
+  def reindex merge = true
```

Comment:
I'm not thrilled with this, but for really large collections of objects, I needed a way to send individual updates instead of a large array. Maybe that suggests that indexing resources should be treated different than indexing `SolrDocument` objects. Thoughts?